app/src/main/java/org/alphatilesapps/alphatiles/Start.java

```diff
@@ -1548,6 +1608,235 @@ public ArrayList<String> parseWordIntoTiles(String parseMe) {
+                    currentTile = tileHashMap.get(currentTileString);
+                    if (MULTIFUNCTIONS.contains(currentTileString)) { // Discern the type of the current tile
+                        int indexInPreliminaryArray = returnInstanceIndexInPreliminaryParsedWordArray(currentTileString, b, parsedWordArrayPreliminary, wordInLOP);
+                        currentTileType = tileList.getInstanceTypeForMixedTilePreliminary(indexInPreliminaryArray, parsedWordArrayPreliminaryReference, wordInLOP);
+                    } else {
+                        currentTileType = currentTile.tileType;
+                    }
+                    if (currentTileType.equals("LV")) {
+                        vowel += currentTileString;
+                    } else if (currentTileType.equals("V")){
+                        nonCombiningVowelFromPreviousSyllable += currentTileString;
+                    }
+                }
+
+                // Find vowel, diacritic, space, and dash symbols that occur between current and next consonants
+                Tile vowelTile;
+                String vowelTileString;
+                String vowelTileType;
+                String vowelToStartTheNextSyllable = "";
+                int indexOfLastFoundMultitypeVowel = -1;
+                for (int a = currentConsonantIndex + 1; a < nextConsonantIndex; a++) {
+                    currentTileString = parsedWordArrayPreliminary.get(a);
+                    currentTile = tileHashMap.get(currentTileString);
+                    if (MULTIFUNCTIONS.contains(currentTileString)) {
+                        int indexInPreliminaryArray = returnInstanceIndexInPreliminaryParsedWordArray(currentTileString, a, parsedWordArrayPreliminary, wordInLOP);
+                        currentTileType = tileList.getInstanceTypeForMixedTilePreliminary(indexInPreliminaryArray, parsedWordArrayPreliminaryReference, wordInLOP);
+                        if(currentTileType.contains("V")){
+                            indexOfLastFoundMultitypeVowel = indexInPreliminaryArray;
+                        }
+                    } else {
+                        currentTileType = currentTile.tileType;
+                    }
+                    if (currentTileType.equals("AV") || currentTileType.equals("BV") || currentTileType.equals("FV")) { // Prepare to add current AV/BV/FV to vowel-so-far
+                        vowelTile = tileHashMap.get(vowel);
+                        if(!Objects.isNull(vowelTile)){ // Vowel composite so far is parsable as one tile in the tile list
+                            vowelTileString = vowelTile.baseTile;
+                            if (MULTIFUNCTIONS.contains(vowelTileString)) { // Discern type of current vowel composite
+                                int indexInPreliminaryArray = returnInstanceIndexInPreliminaryParsedWordArray(vowelTileString, indexOfLastFoundMultitypeVowel, parsedWordArrayPreliminary, wordInLOP);
```

Comment:
FYI, if you haven't already, you may need to make your own test case for discerning a composite. When I put a break point at line 1727, it didn't trigger for any of the two Thai or the Lao language packs.